dataset-processor-python/data_loader.py

```diff
@@ -58,5 +269,11 @@ def create_dataset_folder(self):
     else:
         file_path = input("Enter the file path: ")
     dataset_name = os.path.splitext(os.path.basename(file_path))[0]
-    manager = DatasetWriter(file_path, dataset_name)
-    manager.read_csv()
+    input_handler = UserInputHandler()
+    inputs = input_handler.inputs
+    if not inputs:
+        print("Please enter valid inputs.")
+        sys.exit(1)
+    writer = DatasetWriter(file_path, dataset_name, inputs)
```

Comment:
why not have file_path come from the UserInputHandler?

Reply:
same for dataset_name

Reply:
Good point, moved them to the UserInputHandler!